schema/bom-1.4-SNAPSHOT.proto

```diff
@@ -450,3 +452,179 @@ message Evidence {
+  optional string description = 7;
+  // If available, an in-depth description of the vulnerability as provided by the source organization. Details often include examples, proof-of-concepts, and other information useful in understanding root cause.
+  optional string detail = 8;
+  // Recommendations of how the vulnerability can be remediated or mitigated.
+  optional string recommendation = 9;
+  // Published advisories of the vulnerability if provided.
+  repeated Advisory advisories = 10;
+  // The date and time (timestamp) when the vulnerability record was created in the vulnerability database.
+  optional google.protobuf.Timestamp created = 11;
+  // The date and time (timestamp) when the vulnerability record was first published.
+  optional google.protobuf.Timestamp published = 12;
+  // The date and time (timestamp) when the vulnerability record was last updated.
+  optional google.protobuf.Timestamp updated = 13;
+  // Individuals or organizations credited with the discovery of the vulnerability.
+  optional VulnerabilityCredits credits = 14;
+  // The tool(s) used to identify, confirm, or score the vulnerability.
+  repeated Tool tools = 15;
+  // An assessment of the impact and exploitability of the vulnerability.
+  optional VulnerabilityAnalysis analysis = 16;
+  // affects
+  repeated VulnerabilityAffects affects = 17;
+}
+
+message VulnerabilityReference {
+  // An identifier that uniquely identifies the vulnerability.
+  optional string id = 1;
+  // The source that published the vulnerability.
+  optional Source source = 2;
+}
+
+message VulnerabilityRating {
+  // The source that calculated the severity or risk rating of the vulnerability.
+  optional Source source = 1;
+  // The numerical score of the rating.
+  optional double score = 2;
+  // Textual representation of the severity that corresponds to the numerical score of the rating.
+  optional Severity severity = 3;
+  // Specifies the severity or risk scoring methodology or standard used.
+  optional ScoreMethod method = 4;
+  // Textual representation of the metric values used to score the vulnerability.
+  optional string vector = 5;
+  // An optional reason for rating the vulnerability as it was.
+  optional string justification = 6;
+}
+
+enum Severity {
+  SEVERITY_CRITICAL = 0;
+  SEVERITY_HIGH = 1;
+  SEVERITY_MEDIUM = 2;
+  SEVERITY_LOW = 3;
+  SEVERITY_INFO = 4;
+  SEVERITY_NONE = 5;
+  SEVERITY_UNKNOWN = 6;
+}
+
+enum ScoreMethod {
+  // Common Vulnerability Scoring System v2 - https://www.first.org/cvss/v2/
+  SCORE_METHOD_CVSSV2 = 0;
+  // Common Vulnerability Scoring System v3 - https://www.first.org/cvss/v3-0/
+  SCORE_METHOD_CVSSV3 = 1;
+  // Common Vulnerability Scoring System v3.1 - https://www.first.org/cvss/v3-1/
+  SCORE_METHOD_CVSSV31 = 2;
+  // OWASP Risk Rating Methodology - https://owasp.org/www-community/OWASP_Risk_Rating_Methodology
+  SCORE_METHOD_OWASP = 3;
+  // Other scoring method
+  SCORE_METHOD_OTHER = 4;
+}
```

Comment:
If might be useful, similar reasons to above, to include SCORE_METHOD_NULL as the default 0 value.

Reply:
Corrected in 7a7d7ada967bf138be476c68c1072c2f0696bc78